standard/feature_flag.lua

```diff
@@ -51,23 +51,27 @@ function FeatureFlag.get(flag)
 end
 
 ---@param flag string
----@return boolean
+---@return boolean|string
 function FeatureFlag._get(flag)
 	local config = FeatureFlag.getConfig(flag)
 	return Logic.nilOr(
 		Logic.readBoolOrNil(mw.ext.VariablesLua.var('feature_' .. flag)),
+		mw.ext.VariablesLua.var('feature_' .. flag),
```

Comment:
Might be needed given line 76?
```suggestion
		Logic.nilIfEmpty(mw.ext.VariablesLua.var('feature_' .. flag)),
```

Reply:
or make it Logic.emptyOr instead of nilOr